### PR TITLE
HCAP-1448 - HA/Employer can edit graduation date

### PR DESCRIPTION
**Pull request by mmumby**
## Work completed
- Removed disabled on graduation date field
- Allow HA/Employer to edit status
- Disabled "Update status" button if employer and the cohort has not been assigned.
- Removed date restrictions - HA/Employer can set anything, form is populated with cohort endDate.
- Added `CheckPermission` to `Edit Info`as only MOH/Super user can edit participant details. (removed disable flag)

### CODE REVIEW

File: client/src/components/participant-details/track-graduation.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```
Comment (fw-noel) [inline client/src/components/participant-details/track-graduation.js:1]: Necessary?

File: client/src/components/modal-forms/ManageGraduationForm.js [hunk 2]
```diff
@@ -118,11 +106,6 @@ export const ManageGraduationForm = ({ initialValues, onClose, onSubmit, cohortE
                       </MuiAlert>
                     </Box>
                   )}
-                {isGraduatingBeforeCohortEndDate && (
-                  <MuiAlert severity='warning'>
```
Comment (fw-noel) [inline client/src/components/modal-forms/ManageGraduationForm.js:122]: I think I said to keep this?

File: client/src/components/participant-details/track-graduation.js (hunk 1 above)
Comment (rhattie-ey) [reply client/src/components/participant-details/track-graduation.js:1]: General purpose eslint-disable should *never* make it into master. This was probably just left over from testing.

Comment (rhattie-ey) [reply client/src/components/participant-details/track-graduation.js:1]: It's probably left over from testing. Should not make it into master, though.

**Review by rhattie-ey — COMMENTED** [review]
Noel has already brought up the only concerns I have. Otherwise looks good. Once his comments are addressed, I'll approve this.

File: client/src/components/modal-forms/ManageGraduationForm.js (hunk 2 above)
Comment (mmumby) [reply client/src/components/modal-forms/ManageGraduationForm.js:122]: Oh I misread that message, I'll add this back!

**Review by fw-noel — APPROVED** [review]

**Review by rhattie-ey — APPROVED** [review]

**Pull request merged**